src/patterns/components/awards/primary.hbs

```diff
@@ -4,7 +4,7 @@ order: 1
 hasAngularCode: true
 hasAngularCodeInfo: true
 ---
-<div class="sprk-o-CenteredColumn sprk-o-Stack sprk-o-Stack--medium">
+<div class="sprk-o-CenteredColumn sprk-o-Stack sprk-o-Stack--medium sprk-o-Stack--center-column" data-id="award-1">
```

Comment:
Was the `sprk-o-Stack--center-column` class missing and this addition is a bug fix?